src/HighlightCeption.php

```diff
@@ -63,8 +64,8 @@ public function _before(TestInterface $test)
 
         $this->webDriverModule = $this->getModule($this->config['module']);
         $this->webDriver = $this->webDriverModule->webDriver;
+        $this->cssStyle = "background-color: {$this->config['cssStyle']['background-color']}; color: {$this->config['cssStyle']['color']};";
```

Comment:
Why is locked css style only for "background-color" and text "color"?
It is not flexible. When I want to add more/other styles, I can not.

Reply:
Forgot about this functionality. Added it back into the file. Thanks for pointing that out.